rook/utils/average_utils.py

```diff
@@ -11,3 +11,20 @@ def run_average_by_dim(args):
 
     result = average_over_dims(**args)
     return result.file_uris
+
+
+def run_average_by_shape(args):
+    from daops.ops.average import average_shape
+
+    result = average_shape(**args)
+    return result.file_uris
+
+
+def run_weighted_average(args):
+    from daops.ops.average import average_over_dims
+
+    args["apply_fixes"] = False
+    args["dims"] = ["latitude", "longitude"]
+
+    result = average_over_dims(**args)
+    return result.file_uris
```

Comment:
I suspect this is an artifact from a merge and should be removed. Please confirm.

Reply:
yes 👍 

the used operator is in anther module `rook.utils.weighted_average_utils`